### PR TITLE
Set copyright holder consistently

### LINKED ISSUE #2787: Set copyright holder to Modelica Association everywhere
@GallLeo Can you please clarify the MSL copyright holders once the CLAs have been signed.

@hubertus65 @MartinOtter FYI

### PR DESCRIPTION
**Pull request by beutlich**
Set copyright holder to `Modelica Association and contributors` consistently

close #2787

### CODE REVIEW

**Comment by GallLeo** [comment]
Should the same be applied to ModelicaStandardLibrary/Modelica/Resources/C-Sources?

**Comment by beutlich** [comment]
> 
> 
> Should the same be applied to ModelicaStandardLibrary/Modelica/Resources/C-Sources?

Yes, done in all places.

File: Modelica/Electrical/Analog/Sources.mo [hunk 1]
```diff
@@ -2264,11 +2264,8 @@ Christoph Clau&szlig;
 <strong>Copyright:</strong>
 </dt>
 <dd>
-Copyright &copy; 1998-2018, Modelica Association and Fraunhofer-Gesellschaft.<br>
-<em>The Modelica package is <strong>free</strong> software; it can be redistributed and/or modified
-under the terms of the <strong>Modelica license</strong>, see the license conditions
-and the accompanying <strong>disclaimer</strong> in the documentation of package
-Modelica in file \"Modelica/package.mo\".</em>
+Copyright &copy; 1998-2018, Modelica Association and contributors<br>
+<em>This Modelica package is <u>free</u> software and the use is completely at <u>your own risk</u>; it can be redistributed and/or modified under the terms of the 3-Clause BSD license. For license conditions (including the disclaimer of warranty) visit <a href=\"https://modelica.org/licenses/modelica-3-clause-bsd\">https://modelica.org/licenses/modelica-3-clause-bsd</a>.</em>
```
Comment (dietmarw) [inline Modelica/Electrical/Analog/Sources.mo]: This should be removed as well.

File: Modelica/Electrical/Analog/Sensors.mo [hunk 2]
```diff
@@ -289,11 +289,8 @@ Christoph Clau&szlig;
...
 </dt>
 <dd>
-Copyright &copy; 1998-2018, Modelica Association and Fraunhofer-Gesellschaft.<br>
-<em>The Modelica package is <strong>free</strong> software; it can be redistributed and/or modified
-under the terms of the <strong>Modelica license</strong>, see the license conditions
-and the accompanying <strong>disclaimer</strong> in the documentation of package
-Modelica in file \"Modelica/package.mo\".</em>
+Copyright &copy; 1998-2018, Modelica Association and contributors<br>
+<em>This Modelica package is <u>free</u> software and the use is completely at <u>your own risk</u>; it can be redistributed and/or modified under the terms of the 3-Clause BSD license. For license conditions (including the disclaimer of warranty) visit <a href=\"https://modelica.org/licenses/modelica-3-clause-bsd\">https://modelica.org/licenses/modelica-3-clause-bsd</a>.</em>
 </dd>
```
Comment (dietmarw) [inline Modelica/Electrical/Analog/Sensors.mo]: This should be removed as well.

File: Modelica/StateGraph.mo [hunk 3]
```diff
@@ -3035,7 +3035,7 @@ are nearly always needed).
 </p>
 
 <p>
-Copyright &copy; 1998-2018, Modelica Association and DLR
+Copyright &copy; 1998-2018, Modelica Association and contributors
 </p>
 <p>
 <em>This Modelica package is <u>free</u> software and the use is completely at <u>your own risk</u>; it can be redistributed and/or modified under the terms of the 3-Clause BSD license. For license conditions (including the disclaimer of warranty) visit <a href=\"https://modelica.org/licenses/modelica-3-clause-bsd\">https://modelica.org/licenses/modelica-3-clause-bsd</a>.</em>
```
Comment (dietmarw) [inline Modelica/StateGraph.mo]: remove this

File: Modelica/Thermal/FluidHeatFlow.mo [hunk 4]
```diff
@@ -3124,7 +3124,7 @@ Since mixing may occur, the outlet temperature may be different from the connect
 Outlet temperature is defined by variable T of the corresponding component.</li>
 </ul>
 <p>
-Copyright &copy; 1998-2018, Modelica Association, Anton Haumer, Christian Kral and AIT.
+Copyright &copy; 1998-2018, Modelica Association and contributors
 </p>
 <p>
 <em>This Modelica package is <u>free</u> software and the use is completely at <u>your own risk</u>; it can be redistributed and/or modified under the terms of the 3-Clause BSD license. For license conditions (including the disclaimer of warranty) visit <a href=\"https://modelica.org/licenses/modelica-3-clause-bsd\">https://modelica.org/licenses/modelica-3-clause-bsd</a>.</em>
```
Comment (dietmarw) [inline Modelica/Thermal/FluidHeatFlow.mo]: remove this

File: Modelica/Thermal/HeatTransfer.mo [hunk 5]
```diff
@@ -3333,7 +3333,7 @@ Hubertus Tummescheit, Lund Institute of Technology, Lund, Sweden.
 </p>
   </dd>
 </dl>
-<p><strong>Copyright &copy; 2001-2018, Modelica Association, Michael Tiller and DLR.</strong></p>
+<p><strong>Copyright &copy; 2001-2018, Modelica Association and contributors</strong></p>
 
 <p>
 <em>This Modelica package is <u>free</u> software and the use is completely at <u>your own risk</u>; it can be redistributed and/or modified under the terms of the 3-Clause BSD license. For license conditions (including the disclaimer of warranty) visit <a href=\"https://modelica.org/licenses/modelica-3-clause-bsd\">https://modelica.org/licenses/modelica-3-clause-bsd</a>.</em>
```
Comment (dietmarw) [inline Modelica/Thermal/HeatTransfer.mo]: remove this

**Pull request merged**